Meteor/LIMRAD94.yaml

```
@@ -0,0 +1,40 @@
+plugins:
+  source:
+    - module: intake_xarray
+
+sources:
+  low_res:
+    description: daily 30m 30s averaged radar reflectivity
+    driver: opendap
+    args:
+      auth: null
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/SHIPS/RV-METEOR/cloudradar/low_res_reflectivity_ncfiles/RV-METEOR_LIMRAD94_Ze_{{date.strftime("%Y%m%d")}}.nc
```

The `low_res` dataset is available in another folder:
```suggestion
      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/SHIPS/RV-METEOR/cloudradar/ncfiles/RV-METEOR_LIMRAD94_Ze_{{date.strftime("%Y%m%d")}}.nc
```